test/org/apache/commons/httpclient/HttpMethodBaseUnitTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2010 psiinon@gmail.com
```

Comment:
The year should be changed to current year (I guess the authors too to "The ZAP Development Team"?).